travis.sh

```diff
@@ -93,6 +93,8 @@ function run_docker() {
         -e CXX=${CXX_FOR_BUILD:-${CXX:-c++}} \
         -e CFLAGS \
         -e CXXFLAGS \
+        -e CCACHE_SLOPPINESS \
```

Comment:
[ccache has a default cache size of 5GB](https://ccache.dev/manual/latest.html#_cache_size_management). Did you check whether these extra options have a measurable effect? If not, I suggest to drop them to reduce complexity.

Reply:
It was only using 500mb because the default size on travis is 500mb.